apiary.apib

```diff
@@ -3461,3 +3452,45 @@ This endpoint allows the unspiking of content
             "version_creator": "5481aab74ee1f927e56944cc",
             "versioncreated": "2014-12-11T14:01:27+0000"
         }
+
+# Group Publishing Content
+
+## Publishing a content item [/archive/publish/{_id}]
```

Comment:
can we make it only `/publish` ? this archive should be content at some point so I think it would be better without this